### PR TITLE
Update custodial-card-issuing-integration.mdoc

### PR DESCRIPTION
**Pull request by gshannon-immersve**
changes to guide as per https://www.notion.so/immersve/Errors-in-Custodial-Card-Issuing-Integration-Guide-1351d446ed8a80cc8001d870b95afa06?pvs=4

Your description here.

Ticket Link: _link_here_

### CODE REVIEW

File: imsv-docs-astro/src/content/docs/guides/card-issuing-apps/custodial-card-issuing-integration.mdoc [hunk 1]
```diff
@@ -92,16 +83,30 @@ Resource identifiers are not shared between the live and test environments.
...
+### Register a Billing Funding Channel
+
+You will need a billing funding channel per token and chain.
+See the partner setup instructions in the {% link page="guides/custodial-funding-protocol" /%} guide.
+
+### Fund Your Billing Funds Pool
+
+You need to deposit funds that will be used to settle cardholder transactions.
+
+### Register a Card-funding Funding Channel
```
Comment (ncjones) [inline imsv-docs-astro/src/content/docs/guides/card-issuing-apps/custodial-card-issuing-integration.mdoc:103]:
```suggestion
### Register a Card-Funding Funding Channel
```

File: imsv-docs-astro/src/content/docs/guides/card-issuing-apps/custodial-card-issuing-integration.mdoc [hunk 2]
```diff
@@ -92,16 +83,30 @@ Resource identifiers are not shared between the live and test environments.
...
+You will need a billing funding channel per token and chain.
+See the partner setup instructions in the {% link page="guides/custodial-funding-protocol" /%} guide.
+
+### Fund Your Billing Funds Pool
+
+You need to deposit funds that will be used to settle cardholder transactions.
+
+### Register a Card-Funding Funding Channel
+
+You will need a card-funding funding channel per token and chain. This funding channel will be used with every cardholder funding source created, as corresponds to token and chain, as per the Per Cardholder section, below.
```
Comment (gshannon-immersve) [inline imsv-docs-astro/src/content/docs/guides/card-issuing-apps/custodial-card-issuing-integration.mdoc:105]:
```suggestion
This funding channel will be used with every cardholder funding source created, as corresponds to token and chain, as per the Per Cardholder section, below.
```

File: imsv-docs-astro/src/content/docs/guides/card-issuing-apps/custodial-card-issuing-integration.mdoc [hunk 3]
```diff
@@ -92,16 +83,30 @@ Resource identifiers are not shared between the live and test environments.
...
-You can avoid presenting the card onboarding journey if the user's region
-is not yet available. Call the {% link endpoint="get-supported-regions" /%}
-endpoint to get the current list of available regions for your application.
+You can confirm that all target regions for your application are supported with the {% link endpoint="get-supported-regions" /%}
+endpoint.
 See {% link page="guides/regions" /%} for more details.
 
+### Register a Billing Funding Channel
+
+You will need a billing funding channel per token and chain.
```
Comment (gshannon-immersve) [inline imsv-docs-astro/src/content/docs/guides/card-issuing-apps/custodial-card-issuing-integration.mdoc:96]:
```suggestion
You will need a billing funding channel per token and chain, with a corresponding funding source. (see below).
```

File: imsv-docs-astro/src/content/docs/guides/card-issuing-apps/custodial-card-issuing-integration.mdoc [hunk 4]
```diff
@@ -92,16 +83,30 @@ Resource identifiers are not shared between the live and test environments.
...
+See the partner setup instructions in the {% link page="guides/custodial-funding-protocol" /%} guide.
+
```
Comment (gshannon-immersve) [inline imsv-docs-astro/src/content/docs/guides/card-issuing-apps/custodial-card-issuing-integration.mdoc:98]:
```suggestion

### Register Cardholder Funding Source
You must create a billing Funding Source for per token and chain. This funding source is associated with the "On-Chain Pooled Float", for that token and chain, as per the {% link page="guides/custodial-funding-protocol" /%} guide.

```

**Review by gshannon-immersve — COMMENTED** [review]
Changes as per Nathan's feedback

File: imsv-docs-astro/src/content/docs/guides/card-issuing-apps/custodial-card-issuing-integration.mdoc [hunk 5]
```diff
@@ -92,16 +83,33 @@ Resource identifiers are not shared between the live and test environments.
...
-You can avoid presenting the card onboarding journey if the user's region
-is not yet available. Call the {% link endpoint="get-supported-regions" /%}
-endpoint to get the current list of available regions for your application.
+You can confirm that all target regions for your application are supported with the {% link endpoint="get-supported-regions" /%}
+endpoint.
 See {% link page="guides/regions" /%} for more details.
 
+### Register a Billing Funding Channel
+
+You will need a billing funding channel per token and chain, with a corresponding funding source. (see below).
```
Comment (gshannon-immersve) [inline imsv-docs-astro/src/content/docs/guides/card-issuing-apps/custodial-card-issuing-integration.mdoc:96]:
```suggestion
You will need a billing funding channel per token and chain, with a corresponding funding source (see Register a Billing Funding Source, below).
```

File: imsv-docs-astro/src/content/docs/guides/card-issuing-apps/custodial-card-issuing-integration.mdoc [hunk 6]
```diff
@@ -92,16 +83,33 @@ Resource identifiers are not shared between the live and test environments.
...
+See the partner setup instructions in the {% link page="guides/custodial-funding-protocol" /%} guide.
+
+### Register Cardholder Funding Source
```
Comment (gshannon-immersve) [inline imsv-docs-astro/src/content/docs/guides/card-issuing-apps/custodial-card-issuing-integration.mdoc:99]:
```suggestion
### Register Billing Funding Source
```

File: imsv-docs-astro/src/content/docs/guides/card-issuing-apps/custodial-card-issuing-integration.mdoc (hunk 5 above)
Comment (gshannon-immersve) [inline imsv-docs-astro/src/content/docs/guides/card-issuing-apps/custodial-card-issuing-integration.mdoc:96]:
```suggestion
You will need a billing funding channel per token and chain, with a corresponding funding source (see Register a Billing Funding Source, below).
```

**Pull request closed without merging**